docs/dice_verification.md

```diff
@@ -1,40 +1,206 @@
+
+Follow these menu entries in SeedSigner:<br>
+> Export Xpub --> Single Sig --> Native Segwit --> Sparrow<br>
+
+<img src="img/dicedoc/sesi_export_xpub_2.png" width="600">
+
+Click 'Export Xpub' and SeedSigner will show an animated QR code to be scanned in Sparrow Wallet (where we are still in the wallet creation).<br>
+Scan the QR code SeedSigner is showing in Sparrow Wallet.<br><br>
+The wallet has now been created in Sparrow. Click 'Apply' button to finalize. The wallets settings screen now looks like this:<br>
+<kbd><img src="img/dicedoc/sparrow_wallet_2.png"></kbd>
+
+We will later use this to verify: (1) fingerprint, (2) zpub (click this button to switch between xpub and zpub!) and (3) addresses on the 'Addresses' tab.
+<br><br><br>
+
+## Verifying with Ian Coleman BIP39 website
+
+Go to https://iancoleman.io/bip39 and check 'Show entropy details' (1):<br>
+<kbd><img src="img/dicedoc/coleman_entropy.png"></kbd>
+
+<br>
+Make sure to check (1) 'Hex' and (2) '24 Words' as 'Mnemonic Length'.<br>
+(Do not use 'dice' format because dice 6 will be replaced by 0).<br>
+Then enter the 99 dices numbers in (3). The corresponding seed words are shown in (4):<br>
+<kbd><img src="img/dicedoc/coleman_verify.png"></kbd>
+<br><br>
+The 24 seed words are the same in SeedSigner and the Ian Coleman tool.
+<br><br>
+
+### Verification of (1) fingerprint, (2) zpub and (3) generated addresses
+**Fingerprint:**<br>
+Fingerprint is not shown in the Ian Coleman tool (so cannot be verified here)
+<br><br>
+**Zpub:**<br>
+Scroll down to the 'Derivation Path' section, click on the 'BIP84' tab (1) and find the zpub in (2):
+<kbd><img src="img/dicedoc/coleman_zpub.png"></kbd><br><br>
+Compare to zpub in Sparrow:<br>
+<kbd><img src="img/dicedoc/sparrow_zpub.png"></kbd>
+<br>
+
+Zpub is the same as shown in SeedSigner and Sparrow wallet.
+<br><br>
+**Addresses:**<br>
+Scroll down to the 'Derived Addresses' section and compare the receive addresses to the ones generated in Sparrow ('Addresses' tab of the wallet):
+<kbd><img src="img/dicedoc/coleman_addresses.png"></kbd>
+<br>
+
+Check that the receive addresses all match.<br><br>
+To verify the change addresses change 'External / Internal' to 1 (1):
+<kbd><img src="img/dicedoc/coleman_change_addresses_1.png"></kbd><br><br>
+Compare the change addresses to the ones generated in Sparrow ('Addresses' tab of the wallet):
+<kbd><img src="img/dicedoc/coleman_change_addresses_2.png"></kbd>
+<br>
+
+Check that the change addresses all match.
+<br><br><br>
+
+## Verifying with Seed Tool website
+
+Go to https://bitcoiner.guide/seed/ and click on 'Seed Generation Input' (1):<br>
+<kbd><img src="img/dicedoc/seedtool_1.png"></kbd>
+
+Then click on the 'Show the Entropy Section' tab (1):<br>
+<kbd><img src="img/dicedoc/seedtool_2.png"></kbd>
+
+Enter the 99 dice numbers in (1), in (2) change back to 'Hex', check that (3) is still '24 Words' and the calculated seed words are shown in (4):
```

Comment:
for some reason, this has very strange wrapping, where "(4):" falls half-way down the left side of the image.  Not sure what the fix is.

Reply:
Hm, now not quite sure what you mean here...that's how it look here:
![grafik](https://github.com/SeedSigner/seedsigner/assets/7389834/6cf06318-541b-4032-b0cc-a6654aef70d8)